docs/configuration.md

```diff
@@ -192,19 +192,20 @@ end up paying more then would probably have been necessary.
 
 ### Understand order_types
 
-The `order_types` configuration parameter contains a dict mapping order-types to
-market-types as well as stoploss on or off exchange type and stoploss on exchange
-update interval in seconds. This allows to buy using limit orders, sell using
+The `order_types` configuration parameter maps actions (`buy`, `sell`, `stoploss`) to order-types (`market`, `limit`, ...) as well as configures stoploss to be on the exchange and stoploss on exchange update interval in seconds.
+
+This allows to buy using limit orders, sell using
 limit-orders, and create stoploss orders using market. It also allows to set the
 stoploss "on exchange" which means stoploss order would be placed immediately once
 the buy order is fulfilled. In case stoploss on exchange and `trailing_stop` are
-both set, then the bot will use `stoploss_on_exchange_interval` to check it periodically
-and update it if necessary (e.x. in case of trailing stoploss).
-This can be set in the configuration file or in the strategy.
-Values set in the configuration file  overwrites values set in the strategy.
+both set, then the bot will use `stoploss_on_exchange_interval` to check the stoploss periodically
```

Comment:
"to check the stoploss conditions periodically" ???

I'm not sure for this change...

Reply:
I've reworded this sentence ... hope it's clearer now.

(basically it pulls trailing stoploss up every `interval` seconds if needed).

Reply:
Excellent! 👍